e2e-olm.sh

```diff
@@ -85,12 +76,15 @@ $CSV
 $PKG
 EOF
 
+CSV_CHANNEL=$(sed -nr 's,.*name: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+CURRENT_CSV=$(sed -nr 's,.*currentCSV: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+CSV_NAME=$(echo ${CURRENT_CSV} | cut -d. -f1)
+
 oc create -n $TEST_NAMESPACE -f /tmp/configmap.yaml
 oc create -n $TEST_NAMESPACE -f catalogsource.yaml
 oc create -n $TEST_NAMESPACE -f operatorgroup.yaml
-oc create -n $TEST_NAMESPACE -f subscription.yaml
+oc process -f subscription.yaml -p CSV_NAME=${CSV_NAME} -p STARTING_CSV=${CURRENT_CSV} -p CHANNEL=${CSV_CHANNEL} | oc create -n $TEST_NAMESPACE -f -
```

Comment:
I think this should be OK for now, but at some point I want to come back and make this kube agnostic and not assume templates exist just OLM.

Reply:
:+1: